elasticsearch_exporter.go

```diff
@@ -335,41 +94,43 @@ type Exporter struct {
 
 // NewExporter returns an initialized Exporter.
 func NewExporter(uri string, timeout time.Duration) *Exporter {
-	counters := make(map[string]*prometheus.CounterVec)
-	gauges := make(map[string]*prometheus.GaugeVec)
+	counters := make(map[string]*prometheus.CounterVec, len(counterMetrics)+len(counterVecMetrics))
+	gauges := make(map[string]*prometheus.GaugeVec, len(gaugeMetrics)+len(gaugeVecMetrics))
 
 	for name, info := range counterVecMetrics {
-		log.Printf("Registering %s", name)
+		log.Println("Registering", name)
 		counters[name] = prometheus.NewCounterVec(prometheus.CounterOpts{
 			Namespace: namespace,
 			Name:      name,
 			Help:      info.help,
-		}, append([]string{"cluster", "node"}, info.labels...))
```

Comment:
Hrm.. I assume a node can be part in multiple clusters? If not, I agree that this should be removed as well.